topdown/http_test.go

```diff
@@ -2159,7 +2159,7 @@ func TestNewInterQueryCacheValue(t *testing.T) {
 		Body:       io.NopCloser(bytes.NewBuffer(b)),
 	}
 
-	result, err := newInterQueryCacheValue(BuiltinContext{}, response, b, &forceCacheParams{})
+	result, _, err := newInterQueryCacheValue(BuiltinContext{}, response, b, &forceCacheParams{})
```

Comment:
It would be good if we add some tests which exercise the new optional params.

Reply:
@ashutosh-narkar Not sure I got this right, if you are talking about the new cache config params, we have unit tests in `cache_test.go` exercising them already. 
I have updated usage of `NewInterQueryCacheWithContext` and `InsertWithExpiry` in `http_tests.go` since `http.go` is using it. Let me know if there is anything more to test in http.

Reply:
I mean in the tests in `http_test.go` we invoke `http.send` with its params. I was suggesting to add a case where we call `http.send` with the new params.